docs/source/changes.rst

```diff
@@ -13,3 +13,23 @@ v1.9
    backwards compatibility. However, this function should not be used for comparing term-type like
    it has been used previously. Use :py:func:`PauliTerm.operations_as_set()` instead. In the future,
    ``sort_ops`` will default to False and will eventually be removed.
+ - :py:func:`Program.alloc()` has been deprecated. Please instantiate :py:class:`QubitPlaceholder()`
+   directly or request a "register" (list) of ``n`` placeholders by using the class constructor
+   :py:func:`QubitPlaceholder.register(n)`.
```

Comment:
Could you say where QubitPlaceholder lives here, e.g. `pyquil.quilatom.QubitPlaceholder`.

Reply:
My secret eventual goal is to provide some top-level imports cc #302 so this won't be as hard to find